### DISCUSSION TITLE
Price provider - problem to collect data?

### DISCUSSION THREAD

**Discussion by moritzwietersheim** [cryptoadvance/specter-desktop#805] [Specter UI/UX]
I have problems collecting price data form providers.

Coindesk works.

If Specter is collecting through "Tor", do i need to run Tor? Probably yes...

"Failed to fetch price data from selected provider, please try again or select another provider."

Just checked the Telegram channel and couldnt find the error message there.

**Comment by ben-kaufman** [comment]
This means something went wrong getting the price, could be anything though.
For over Tor, yes you need to have Tor working. Also, it already happened in the past that a provider simply was down, and that was the issue. So most likely either that or a Tor issue.
